src/ContentBundle/Model/Content.php

```diff
@@ -849,7 +849,9 @@ public function getCoverImage()
         if ($this->getValueSet() !== null) {
             foreach ($this->getValueSet()->getValues() as $value) {
                 if ($value instanceof MediaValue && null !== $media = $value->getMedias()->first()) {
-                    return $media->getReference();
+                    if(is_object($media)) {
```

Comment:
Can you change this into an `instanceof MediaInterface` check?

(And please add an empty space between `if` and `(`)